miner/proposal_builder.go

```diff
@@ -74,7 +74,11 @@ type ProposalBuilder struct {
 
 	mu      sync.Mutex
 	signers map[types.NodeID]*signerSession
-	shared  sharedSession
```

Comment:
was it necessary to move it?

Reply:
I didn't, it's still there 🙂. It just moved a bit further down when I added the fields for the active set.

Reply:
yes i meant to ask, why it was necessary to move field by a couple of lines. seems like a random change

Reply:
I wanted to group fields that belong to the same mutex, but I can also do this with a `struct` as you suggested.